fbw-a380x/src/wasm/systems/a380_systems/src/hydraulic/mod.rs

```diff
@@ -986,20 +940,10 @@ impl A380RudderFactory {
             || context.start_state() == StartState::Runway
             || context.is_in_flight();
 
-        let upper_assembly = Self::a380_rudder_assembly(
-            context,
-            init_at_center,
-            true,
-            Self::UPPER_AND_LOWER_PANEL_UPPER_EBHA_BUS,
-            Self::UPPER_PANEL_LOWER_EBHA_BUS,
-        );
-        let lower_assembly = Self::a380_rudder_assembly(
-            context,
-            init_at_center,
-            false,
-            Self::UPPER_AND_LOWER_PANEL_UPPER_EBHA_BUS,
-            Self::LOWER_PANEL_LOWER_EBHA_BUS,
-        );
+        let upper_assembly =
```

Comment:
Upper rudder upper EBHA is AC ESS, upper rudder lower EBHA is AC EHA, lower rudder upper EBHA is AC ESS, and lower rudder lower EBHA is AC 1